### DISCUSSION TITLE
Default sort column

### DISCUSSION THREAD

**Discussion by jigapate** [Power-Components/livewire-powergrid#1453] [Q&A]
Hello,

How to define the default column for sorting when the table is loaded? say there is Date column in table, and that is need to to sorted desc while loading table,

Thanks.

**Comment by jigapate** [comment]
Got the answer, :-)

 public string $sortField = 'name';
    
public string $sortDirection = 'desc';

**Comment by whjvdijk** [comment]
@jigapate could you elaborate a bit more. How did you exactly fix this?
